src/routes/Sponsors.svelte

```diff
@@ -179,7 +179,7 @@
 
 		{#if speakerEventSponsors.length > 0}
 			<h3 class="mt-24 w-full text-left">{$_('sponsors.tiers.speakerEvent')}</h3>
-			<div class="grid md:grid-cols-3 items-center">
+			<div class="grid md:grid-cols-4 items-center">
```

Comment:
When we add new logos that don't have the same dimensions, it will break again. I think, a proper fix for the root cause would be to add the `w-full` class to all the `grid` `div`s in sponsors.